api/backup.go

```diff
@@ -76,6 +76,20 @@ func (e BackupStorageBackend) String() string {
 	return "UNKNOWN"
 }
 
+// BinlogInfo is the binlog coordination for MySQL.
+type BinlogInfo struct {
+	FileName string `json:"fileName"`
+	Position int64  `json:"position"`
+}
+
+// BackupPayload is encoded in JSON and stored in the backup table, representing PITR related info.
+type BackupPayload struct {
+	BinlogInfo BinlogInfo `json:"binlogInfo"`
+	// Imprecise UNIX timestamp to the second which is the rough time when this backup is taken.
```

Comment:
With regard to the comment of BinlogInfo, we need to comment on why we still need this extra Ts, since we can already get a timestamp info from that field.

Reply:
Added comments. It's only for UI purposes.

Because finding out the precise timestamp of the binlog position (or later pg WAL lsn) is time/resource consuming, and may involve file downloading and parsing. This `Ts` is just the wall clock time of bytebase when the backup is taken.

Reply:
Second thought: there's a `CreatedTs` in the `api.Backup`, which is the time we created the backup record in db. It's actually not that far from the new `Ts` field, which is the time we actually dump the backup.

Do we still want to distinguish them and add a new `Ts` field? @d-bytebase

Reply:
Removed `Ts`.